### PR TITLE
Change remark to description and update sameperson check

### PR DESCRIPTION
**Pull request by isanidiot**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/UpdateCommand.java [hunk 1]
```diff
@@ -132,10 +132,10 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         MeetingTime updatedMeetingTime = editPersonDescriptor.getMeetingTime().orElse(personToEdit.getMeetingTime());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
         FilePath updatedFilePath = personToEdit.getFilePath();
-        Remark updatedRemark = personToEdit.getRemark(); // edit command does not allow editing remarks
+        Description updatedDescription = personToEdit.getDescription(); // edit command does not allow editing remarks
```
Comment (ReaganTan00) [inline src/main/java/seedu/address/logic/commands/UpdateCommand.java:135]: I think we should enhance UpdateCommand to allow updating of a client's description.
Can implement as a separate PR/Issue

Comment (ReaganTan00) [reply src/main/java/seedu/address/logic/commands/UpdateCommand.java:135]: Current implementation does not allow updating of descriptions

File: src/main/java/seedu/address/model/person/Person.java [hunk 2]
```diff
@@ -131,7 +131,7 @@ public boolean isSamePerson(Person otherPerson) {
 
         return otherPerson != null
                 && otherPerson.getName().equals(getName())
-                && (otherPerson.getPhone().equals(getPhone()) || otherPerson.getEmail().equals(getEmail()));
+                && otherPerson.getPhone().equals(getPhone());
```
Comment (ReaganTan00) [inline src/main/java/seedu/address/model/person/Person.java:134]: Updated samePerson check 👍

File: src/test/java/seedu/address/model/person/PersonTest.java [hunk 3]
```diff
@@ -45,10 +45,10 @@ public void isSamePerson() {
                 .withTags(VALID_TAG_HUSBAND).build();
         assertTrue(ALICE.isSamePerson(editedAlice));
 
-        // same name, same email, different attributes -> returns true
+        // same name, same email, different attributes -> returns false
         editedAlice = new PersonBuilder(ALICE).withPhone(VALID_PHONE_BOB).withAddress(VALID_ADDRESS_BOB)
                 .withTags(VALID_TAG_HUSBAND).build();
-        assertTrue(ALICE.isSamePerson(editedAlice));
+        assertFalse(ALICE.isSamePerson(editedAlice));
```
Comment (ReaganTan00) [inline src/test/java/seedu/address/model/person/PersonTest.java:51]: Updated test due to changed isSamePerson check 👍

**Review by ReaganTan00 — COMMENTED** [review]
LGTM after resolving merge conflicts
To consider:
Allowing updating of description through UpdateCommand

**Review by ReaganTan00 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/UpdateCommand.java (hunk 1 above)
Comment (isanidiot) [reply src/main/java/seedu/address/logic/commands/UpdateCommand.java:136]: right now editting description has its own command

**Pull request merged**